### PR TITLE
updated main.js

### PR DESCRIPTION
**Pull request by TashDevCat**
added a check for short urls and ip grabbers

### CODE REVIEW

**Comment by dominikaaaa** [comment]
I'll review tomorrow, don't merge

**Comment by TashDevCat** [comment]
is confuzed?

**Comment by TashDevCat** [comment]
oh nvm i pushed it from my branch in my fork into my master in my fork

**Comment by humboldt123** [comment]
LGTM. Wait for @dominikaaaa and her official regex seal of approval.

File: main.js [hunk 1]
```diff
@@ -38,7 +38,11 @@ const fs = require("graceful-fs");
 const convert = require("ms")
 const fetch = require("node-fetch")
 
+
 // Regexes
+/*ipLoggers*/
```
Comment (dominikaaaa) [inline main.js:43]: bad formatting rejected pull request

File: commands/historian.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const fs = require("graceful-fs");
 module.exports.run = async (client, message, args) => {
 
     message.channel.send(`I will eat you`);
-    message.delete()
+    return message.delete()
```
Comment (sourTaste000) [inline commands/historian.js:7]: I dont think you need return here

File: commands/hw.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const Discord = require("discord.js");
+const fs = require("graceful-fs");
+
+module.exports.run = async (client, message, args) => {
+        message.channel.send(`<@${message.mentions.users.first().id}>, please move to <#634010851128639508> or <#724349410858893372>`)
+        return message.delete()
```
Comment (sourTaste000) [inline commands/hw.js:6]: here

File: commands/xiaro.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ const fs = require("graceful-fs");
 
 module.exports.run = async (client, message, args) => {
     message.channel.send(`Trans rights~! :3, ${message.content.slice(7)}`)
-    message.delete()
+    return message.delete()
```
Comment (sourTaste000) [inline commands/xiaro.js:6]: and here

File: main.js [hunk 5]
```diff
@@ -274,7 +278,15 @@ async function autoResponder(message) {
     
     /* only moderators bypass */
     if (!message.member.hasPermission("BAN_MEMBERS")) {
+        /*ip logger regex */
         if (!cleanedMessage.length && !message.attachments) return message.delete();
```
Comment (sourTaste000) [inline main.js:282]: you don't return in if statement
you can just do
```js
if(!cleanedMessage.length && !message.attachments) message.delete()
```

File: main.js [hunk 6]
```diff
@@ -274,7 +278,15 @@ async function autoResponder(message) {
...
+        if (grabbifyLoggers.test(cleanedMessage)||
+        new RegExp("linkify.{0,3}me|raboninco.{0,3}com|j.{0,3}mp|bit.{0,3}ly|goo.{0,3}gl|//tinyurl.{0,3}com|is.{0,3}gd|bit.{0,3}do|iplogger.{0,3}org/logger|ps3cfw.{0,3}com/cool.php|rb.{0,3}gy|blasze.{0,3}tk").test(cleanedMessage)||
+        new RegExp("")) {
```
Comment (sourTaste000) [inline main.js:285]: Why are you making a new `RegExp`? Just reply the message

File: commands/historian.js (hunk 2 above)
Comment (dominikaaaa) [reply commands/historian.js:7]: yes you do

File: commands/hw.js (hunk 3 above)
Comment (dominikaaaa) [reply commands/hw.js:6]: yes you do

File: commands/xiaro.js (hunk 4 above)
Comment (dominikaaaa) [reply commands/xiaro.js:6]: yes you do

File: main.js (hunk 5 above)
Comment (dominikaaaa) [reply main.js:282]: yes you can and yes you do

File: main.js (hunk 6 above)
Comment (dominikaaaa) [reply main.js:285]: good question

File: main.js (hunk 5 above)
Comment (TashDevCat) [reply main.js:282]: that was already there i didnt add the return i think

File: main.js (hunk 6 above)
Comment (TashDevCat) [reply main.js:285]: i fixed the regex formatting and deleted the unused regexp in a different pr, the one that adds bigrat.js

File: commands/historian.js [hunk 7]
```diff
@@ -4,7 +4,7 @@ const fs = require("graceful-fs");
 module.exports.run = async (client, message, args) => {
 
     message.channel.send(`I will eat you`);
-    return message.delete()
+    message.delete()
```
Comment (dominikaaaa) [inline commands/historian.js]: revert this, you *need* the return.

File: commands/xiaro.js [hunk 8]
```diff
@@ -3,7 +3,7 @@ const fs = require("graceful-fs");
 
 module.exports.run = async (client, message, args) => {
     message.channel.send(`Trans rights~! :3, ${message.content.slice(7)}`)
-    return message.delete()
+    message.delete()
```
Comment (dominikaaaa) [inline commands/xiaro.js]: revert this, you *need* the return

File: main.js (hunk 6 above)
Comment (dominikaaaa) [reply main.js:285]: bruh why not just push to the same branch to fix it

**Comment by TashDevCat** [comment]
i reverted it, i though i needed the return but than historian and >.< as it says plz let me jump off bridge

**Comment by TashDevCat** [comment]
i also added bigrat.js in this pr and will revoke the other pr

**Comment by humboldt123** [comment]
You could have made smaller prs but it doesn't look like it would break. I'll merge in the morning. 😁

**Pull request merged**